github_connector/wizards/wizard_update_from_github.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class WizardUpdateFromGithub(models.TransientModel):
+    _name = 'wizard.update.from.github'
```

Comment:
I get this warning:
`wizard.update.from.github has no _description`
so I think It's better that you add the parameter
`_description`